src/mobile/mobileVelocityController.js

```diff
@@ -23,3 +23,9 @@ export function setAirfighterVelocityFromMobileJoystick(
     airfighter.element.src = 'img/aifighter-Down.gif'
   }
 }
+
+export function mobilePauseGame() {
```

Comment:
It is weird that `mobilePauseGame` is in `src/mobile/mobileVelocityController.js`, because velocity has no relation pause. Please study philosophy.